doc/contributing.xml

```diff
@@ -6,12 +6,28 @@
 
 <para>The DocBook sources of the Nixpkgs manual are in the <filename
 xlink:href="https://github.com/NixOS/nixpkgs/tree/master/doc">doc</filename>
-subdirectory of the Nixpkgs repository. If you make modifications to
-the manual, it's important to build it before committing. You can do that as follows:
+subdirectory of the Nixpkgs repository.</para>
+
+<para>You can quickly check your edits with <command>make</command>:</para>
+
+<screen>
+  $ cd /path/to/nixpkgs/doc
+  $ nix-shell
+  [nix-shell]$ make
+</screen>
+
+<para>If you experience problems, run <command>xmloscopy --docbook5
+./manual.xml ./manual-full.xml</command> inside the nix-shell to help
```

Comment:
small improvement idea: add a `make debug` shortcut command for that scenario

Reply:
That would run like `nix-shell --run 'xmloscopy ...'`. Does that sound right?

Reply:
Do the other make targets assume a nix-shell environment?

Reply:
its complicated because this is the only one that won't work with the default.nix, where the makefile will be called automatically.

Reply:
I'm going to merge, we can improve this in a separate commit :)